pkgs/tools/archivers/7zz/default.nix

```diff
@@ -109,7 +121,7 @@ stdenv.mkDerivation rec {
       # the unRAR compression code is disabled by default
       lib.optionals enableUnfree [ unfree ];
     maintainers = with maintainers; [ anna328p peterhoeg jk ];
-    platforms = platforms.unix;
+    platforms = platforms.unix ++ platforms.windows;
```

Comment:
Thanks for working on this.

One question though. Is `platforms.windows` correct here?

```
nix-repl> lib.platforms.windows
[ "i686-cygwin" "x86_64-cygwin" "x86_64-windows" "i686-windows" ]
```

I don't know much about windows builds but it seems like Cygwin is a bit different to MinGW
Is `"x86_64-cygwin"` actually under `isMinGW`?

```
isCygwin       = { kernel = kernels.windows; abi = abis.cygnus; };
isMinGW        = { kernel = kernels.windows; abi = abis.gnu; };
```

Reply:
Cygwin and MinGW are indeed very different. I don't think it matters here because `platforms.unix` already includes Cygwin.

Reply:
so it should have stayed `platforms.unix` and not added `platforms.windows`?

Reply:
The two mingw doubles were needed, which `platforms.windows` provides.

Reply:
Your question makes sense. Perhaps there should be a `platforms.mingw`, using that would have made the intention of the change clearer.